.travis.yml

```diff
@@ -4,6 +4,10 @@ compiler:
   - clang
   - gcc
 
+env:
+  - BUILD_TYPE=Release
+  - BUILD_TYPE=Debug
```

Comment:
I believe Gijs has tried this before, but Travis timed out because it takes too long.